easel/lib/utils/constants.dart

```diff
@@ -134,8 +134,8 @@ const kAgoricSymbol = 'urun';
 const kJunoSymbol = 'ujunox';
 const String kEthereumSymbol = "weth-wei";
 
-const kPylonText = 'Pylon';
-const kUSDText = 'USD';
+const kPylonText = 'IAP';
```

Comment:
It doesn't match with the variable name. Create a separate variable for it.

Reply:
done